src/ai/backend/manager/models/vfolder.py

```diff
@@ -130,6 +130,7 @@ class VFolderAccessStatus(str, enum.Enum):
 
     READABLE = "readable"
     UPDATABLE = "updatable"
+    DELETABLE = "deletable"
 
```

Comment:
Please check whether we need additional database migration script or not.

Reply:
we do not need db migration because `VFolderAccessStatus` is not used by any db column